src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -8,6 +8,7 @@ import checkboxStyles from './checkbox.scss';
 type Props = SwitchProps & {
     skin: 'dark' | 'light',
     className?: string,
+    inactive?: boolean,
```

Comment:
How about `active` as a mandatory parameter, and setting it to `true` by default? Positive names are easier to understand IMO.